### DISCUSSION TITLE
VMware vCenter (VCSA) / ESXi Host Monitoring

### DISCUSSION THREAD

**Discussion by shokoah** [amidaware/tacticalrmm#1693] [Ideas]
Just an idea - I am a subscriber and love how TRMM works - I have it installed on Windows, macOS, and Linux agents and use it almost daily. I work a lot with ESXi hosts, I am comparing this mainly to Datto RMM (CentraStage). The idea would be providing a new column on the main page for "Workstations | Servers | Hosts" to add VMware monitoring and simple commands, e.g.:

- Start/Stop VM's
- Enable/Exit Maintenance Mode
- Review datastore usage
- Review host resource usage
- Review VM resource usage

Attached are some screenshots from CentraStage:

![Host List](https://github.com/amidaware/tacticalrmm/assets/32691688/410c5e5a-2f00-446a-a028-d1d11b70cdc8)
![Guests](https://github.com/amidaware/tacticalrmm/assets/32691688/5fbf9f33-5e2e-474f-8b14-dc37f6542e32)
![Monitors and Hardware](https://github.com/amidaware/tacticalrmm/assets/32691688/46fafaf6-fda0-43cf-b781-500556e01df5)
![Storage](https://github.com/amidaware/tacticalrmm/assets/32691688/6877ddcf-062c-4180-b93a-a0ea13063c30)

I am pretty in tune with ESX and would love to see this come true

**Comment by silversword411** [comment]
This is a very niche ask, I wouldn't hold your breath on this one. 

You can probably use the existing Site mechanisms to have a site per esxi host and group that way as a workaround. When we get to column customization we can see about adding the server/workstation as an item as well https://github.com/amidaware/tacticalrmm/issues/308
